backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java

```diff
@@ -320,8 +425,6 @@ public CommitEntity getCommitEntity(
       }
 
       session.beginTransaction();
-      // delete associated branch
```

Comment:
context of this change?

Reply:
there is a check before this code that a commit shouldn't be associated with a branch. So it means that a branch deletion is not needed. cc: @ad-47